.teamcity/_self/projects/WebApp.kt

```diff
@@ -104,7 +104,7 @@ object RunCalypsoE2eDesktopTests : BuildType({
 				export NODE_CONFIG="{\"calypsoBaseURL\":\"${'$'}{URL%/}\"}"
 				export TEST_FILES=${'$'}(join ',' ${'$'}(find specs*/**/*spec.js -type f -not -path specs-playwright/*))
 
-				yarn magellan --config=magellan.json --max_workers=%E2E_WORKERS% --suiteTag=parallel --local_browser=chrome --mocha_args="--reporter mocha-teamcity-reporter" --test=${'$'}{TEST_FILES}
+				yarn magellan --config=magellan.json --max_workers=%E2E_WORKERS% --suiteTag=parallel --local_browser=chrome --test=${'$'}{TEST_FILES}
```

Comment:
As mentioned in the top comment, the `mocha_args` flag here was overriding the flags specified in magellan configs. This would have been acceptable when the mocha configuration file loaded `mocha-hooks.js` by default, but this is problematic when mocha config no longer loads a hook file by default.

Reply:
We do want to pass `--reporter mocha-teamcity-reporter` here because it is a CI-only thing: if someone runs e2e tests in their local machines, we don't want that reporter to be loaded.

Same for all the other flags really. `magellan.json` should define the defaults for running the tests locally, but in CI we override some of those flags to adapt to the environment.